src/Symfony/Bridge/Twig/Node/SearchAndRenderBlockNode.php

```diff
@@ -49,8 +50,15 @@ public function compile(Compiler $compiler): void
                         // of variables at compile time.
                         $labelIsExpression = false;
 
+                        if (method_exists(CoreExtension::class, 'testEmpty')) {
+                            $isEmpty = CoreExtension::testEmpty($label->getAttribute('value'));
+                        } else {
+                            // to be removed when support for Twig 3 is dropped
+                            $isEmpty = twig_test_empty($label->getAttribute('value'));
+                        }
+
                         // Only insert the label into the array if it is not empty
-                        if (!twig_test_empty($label->getAttribute('value'))) {
+                        if (!$isEmpty) {
```

Comment:
Can we use `$label !== null && $label !== ''` as condition instead ? We don't need to handle cases about checking emptiness for other kind of values here.

Reply:
done